timemachine/fe/atom_mapping.py

```diff
@@ -169,10 +178,35 @@ def get_core_by_mcs(
 
     cost = np.zeros((len(matches_a), len(matches_b)))
 
+    chiral_set_a = ChiralRestrIdxSet.from_mol(mol_a, conf_a)
+    chiral_set_b = ChiralRestrIdxSet.from_mol(mol_b, conf_b)
+
+    def valid_chiral_atoms(trial_core):
+        if allow_chiral_atom_flips and allow_chiral_atom_undefined:
+            return True
+
+        flips = find_atom_map_chiral_conflicts(trial_core, chiral_set_a, chiral_set_b, mode="flip")
+        undefineds = find_atom_map_chiral_conflicts(trial_core, chiral_set_a, chiral_set_b, mode="undefined")
+
+        num_flips = len(flips)
+        num_undefineds = len(undefineds)
+
+        # TODO: de-DeMorgan-ify
+        invalid_due_to_flips = (not allow_chiral_atom_flips) and (num_flips > 0)
+        invalid_due_to_undefineds = (not allow_chiral_atom_undefined) and (num_undefineds > 0)
+        invalid = invalid_due_to_flips or invalid_due_to_undefineds
+        valid = not invalid
```

Comment:
lol

Reply:
a bit convoluted, will simplify

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/908/commits/68cbae755248cbd306444fcfda290c45818a5042